curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/65552b14e803731fe3c1e4ca.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: 65552b14e803731fe3c1e4ca
-title: Step 81
+title: Step 85
 challengeType: 20
-dashedName: step-81
+dashedName: step-85
 ---
 
 # --description--
```

Comment:
Modified instruction:
Next, modify your `encryption` and `decryption` variables by calling `encrypt` and `decrypt`, respectively, iin place of `vignere`.